nginx_proxy/start_nginx.sh

```diff
@@ -79,12 +93,40 @@ if [[ -f "${CONF_FILE}" ]]; then
   cp "${CONF_FILE}" /etc/nginx/nginx.conf
 fi
 
-# fetch Service Configuration from Service Management if the service name and
-# service version are provided.
-if [[ -n "${ENDPOINTS_SERVICE_NAME}" && \
-      -n "${ENDPOINTS_SERVICE_VERSION}" ]]; then
-  /usr/sbin/fetch_service_config.sh \
-    -s "${ENDPOINTS_SERVICE_NAME}" -v "${ENDPOINTS_SERVICE_VERSION}" || exit $?
-fi
+# Start nginx
```

Comment:
Please add a comment with more detail here. Some commentary on the options choices (i.e. what it means to be using a service name vs version and rollout strategy) would be helpful.

Reply:
Done